examples/server/server.cpp

```diff
@@ -677,6 +691,14 @@ struct llama_server_context
         slot_params default_params;
         llama_sampling_params default_sparams;
 
+        if (data.count("__oaicompat") != 0) {
+            slot->oaicompat = true;
+            slot->oaicompat_model = json_value(data, "model", std::string(DEFAULT_OAICOMPAT_MODEL));
+        } else {
+            slot->oaicompat = false;
+            slot->oaicompat_model = "";
+        }
```

Comment:
I feel like this business is somewhat redundant. I think it's better if the api returns the `params.model_alias`. I tried a few clients and all of them handle this just fine. In fact it provides better results because it makes it more clear that it's actually talking to the localhost endpoint. 

If someone really needs the server to respond with the exact openai model name then the model_alias parameter to the server already allows you to do that.